### PR TITLE
Studio app availability

### PR DESCRIPTION
**Pull request by thebiglabasky**
Studio App is now available for Windows and Linux too

### CODE REVIEW

**Comment by martzoukos** [comment]
There are more places where this is mentioned. Try a full project search (or I can do it in a couple of hours).

**Comment by thebiglabasky** [comment]
I just pushed this one I stumbled upon so that would be an easy fix, but what you're saying makes sense indeed 😄

**Comment by martzoukos** [comment]
@thebiglabasky I've made the changes and also removed the, now obsolete, troubleshooting section about the macOS security check.

It all looks good to me now, so we can merge if you're 👍

File: content/100-getting-started/01-quickstart.mdx [hunk 1]
```diff
@@ -648,7 +648,7 @@ Prisma Studio is a visual editor for the data in your database.
 You can run it with two ways:
 
 1. Run `$ npx prisma studio` in your terminal.
-2. Install the macOS app from the installers which you can find [here](https://github.com/prisma/studio/releases). Currently, only macOS is supported - if you want to install the native app on Windows or Linux, please let us know by opening an [issue](https://github.com/prisma/studio/issues) in Prisma Studio.
+2. Install the macOS app from the installers which you can find [here](https://github.com/prisma/studio/releases).  Windows, macOS and Linux are supported.
```
Comment (thebiglabasky) [inline content/100-getting-started/01-quickstart.mdx]:
```suggestion
2. Install [the desktop app](https://github.com/prisma/studio/releases) from the installers.  Windows, macOS and Linux are supported.
```

File: content/100-getting-started/02-setup-prisma/01-add-to-existing-project.mdx [hunk 2]
```diff
@@ -794,7 +794,7 @@ Prisma Studio is a visual editor for the data in your database.
 You can run it with two ways:
 
 1. Run `$ npx prisma studio` in your terminal.
-2. Install the macOS app from the installers which you can find [here](https://github.com/prisma/studio/releases). Currently, only macOS is supported - if you want to install the native app on Windows or Linux, please let us know by opening an [issue](https://github.com/prisma/studio/issues) in Prisma Studio.
+2. Install the macOS app from the installers which you can find [here](https://github.com/prisma/studio/releases).  Windows, macOS and Linux are supported.
```
Comment (thebiglabasky) [inline content/100-getting-started/02-setup-prisma/01-add-to-existing-project.mdx]: Some light accessibility: no "here" links as bad for screen readers.
```suggestion
2. Install [the desktop app](https://github.com/prisma/studio/releases) from the installers.  Windows, macOS and Linux are supported.
```

File: content/100-getting-started/02-setup-prisma/02-start-from-scratch.mdx [hunk 3]
```diff
@@ -801,7 +801,7 @@ Prisma Studio is a visual editor for the data in your database.
 You can run it with two ways:
 
 1. Run `$ npx prisma studio` in your terminal.
-2. Install the macOS app from the installers which you can find [here](https://github.com/prisma/studio/releases). Currently, only macOS is supported - if you want to install the native app on Windows or Linux, please let us know by opening an [issue](https://github.com/prisma/studio/issues) in Prisma Studio.
+2. Install the macOS app from the installers which you can find [here](https://github.com/prisma/studio/releases).  Windows, macOS and Linux are supported.
```
Comment (thebiglabasky) [inline content/100-getting-started/02-setup-prisma/02-start-from-scratch.mdx]:
```suggestion
2. Install [the desktop app](https://github.com/prisma/studio/releases) from the installers.  Windows, macOS and Linux are supported.
```

File: content/200-concepts/100-components/06-prisma-studio.mdx [hunk 4]
```diff
@@ -11,23 +11,12 @@ Prisma Studio is a visual editor for the data in your database.
 You can run it with two ways:
 
 1. Run `$ npx prisma studio` in your terminal.
-2. Install the macOS app from the installers which you can find [here](https://github.com/prisma/studio/releases). Currently, only macOS is supported - if you want to install the native app on Windows or Linux, please let us know by opening an [issue](https://github.com/prisma/studio/issues) in Prisma Studio.
-
-> **Note**: Prisma Studio replaces Prisma Admin, and is only compatible with Prisma 2.
+2. Install the desktop app from the installers which you can find [here](https://github.com/prisma/studio/releases). Windows, macOS and Linux are supported.
```
Comment (thebiglabasky) [inline content/200-concepts/100-components/06-prisma-studio.mdx]:
```suggestion
2. Install [the desktop app](https://github.com/prisma/studio/releases) from the installers.  Windows, macOS and Linux are supported.
```

**Review by thebiglabasky — COMMENTED** [review]
Looks good to me with these last changes. I'll let @mhwelander push the button

**Pull request merged**